.bldr.toml

```diff
@@ -63,6 +63,11 @@ plan_path = "bzip2-musl"
 paths = [
   "bzip2/*",
 ]
+[c-ares]
+plan_path = "c-ares"
+paths = [
```

Comment:
For this package I don't think you should actually need the paths = [ "c-ares/*" ] option. In the core-plans repo this is mostly for planfiles that are sourcing other planfiles.

Reply:
Done ;)